### PR TITLE
added home work 1

### PR DESCRIPTION
**Pull request by DmytrV**

### CODE REVIEW

File: DmytroV/HW1/1.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+user_name = input("What is your name? ")
```
Comment (kolyasalubov) [inline DmytroV/HW1/1.py:1]: good job

File: DmytroV/HW1/2.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+user_name = input("What is your name? ")
```
Comment (kolyasalubov) [inline DmytroV/HW1/2.py:1]: good job

File: DmytroV/HW1/1.py (hunk 1 above)
Comment (DmytrV) [reply DmytroV/HW1/1.py:1]: Thank you :)

File: DmytroV/HW1/2.py (hunk 2 above)
Comment (DmytrV) [reply DmytroV/HW1/2.py:1]: Thank you :)

File: DmytroV/HW1/1.py (hunk 1 above)
Comment (kolyasalubov) [reply DmytroV/HW1/1.py:1]: Not at all)!

File: DmytroV/HW1/2.py (hunk 2 above)
Comment (kolyasalubov) [reply DmytroV/HW1/2.py:1]: Not at all)!

File: DmytroV/HW6/Task 1.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+even_numbers = []
```
Comment (kolyasalubov) [inline DmytroV/HW6/Task 1.py:1]: good job

File: DmytroV/HW6/Task 2.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+login = input("Enter a loggin:")
```
Comment (kolyasalubov) [inline DmytroV/HW6/Task 2.py:1]: good job
look at this implementation:
user_name = input('Hello, please input your Log in:')

while user_name != 'First':
    user_name = input('Error: wrong username, please try one more time. Username:')
else:
    print('Greeting. Access granted!!!', user_name)

File: DmytroV/HW3/1.py [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+Unless explicitly silenced.
+In the face of ambiguity, refuse the temptation to guess.
+There should be one-- and preferably only one --obvious way to do it.
+Although that way may not be obvious at first unless you're Dutch.
+Now is better than never.
+Although never is often better than *right* now.
+If the implementation is hard to explain, it's a bad idea.
+If the implementation is easy to explain, it may be a good idea.
+Namespaces are one honking great idea -- let's do more of those!"""
+
```
Comment (kolyasalubov) [inline DmytroV/HW3/1.py:58]: good job

**Pull request merged**